src/vm/ecalllist.h

```diff
@@ -108,7 +108,7 @@ FCFuncStart(gDependentHandleFuncs)
     FCFuncElement("nGetPrimary",             DependentHandle::nGetPrimary)
     FCFuncElement("nGetPrimaryAndSecondary", DependentHandle::nGetPrimaryAndSecondary)
     FCFuncElement("nFree",                   DependentHandle::nFree)
-    FCFuncElement("nSetDependentHandle",     SetDependentHandleSecondary)
+    FCFuncElement("nSetDependentHandleSecondary",     SetDependentHandleSecondary)
```

Comment:
This needs to go via `DependentHandle::nSetSecondary` like all the other ones. It won't have the right calling convention otherwise.